lib/Mojo/Transaction/WebSocket.pm

```diff
@@ -86,6 +86,8 @@ sub parse_message {
   # Append chunk and check message size
   $self->{op} = $op unless exists $self->{op};
   $self->{message} .= $frame->[5];
+  # set pmce (rsv1) from first frame in chain
+  $self->{pmc} = $frame->[1] unless exists $self->{pmc};
```

Comment:
Calling this `pmc` seems wrong when it's not actually per-message compression specific.